doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,322 @@
+
+  * nRF9160:
+
+    * :ref:`secure_services`
+    * :ref:`secure_partition_manager`
+    * :ref:`aws_fota_sample`
+    * :ref:`lte_sensor_gateway`
+
+  * Bluetooth Low Energy:
+
+    * :ref:`central_bas`
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_lbs`
+    * :ref:`shell_bt_nus`
+
+  * Other:
+
+    * :ref:`ppi_trace_sample`
+
+* Added or updated documentation for the following libraries:
+
+  * nRF9160:
+
+    * :ref:`lib_spm`
+    * :ref:`at_cmd_readme`
+    * :ref:`lib_download_client`
+    * :ref:`lib_aws_fota`
+    * :ref:`lib_fota_download`
+    * :ref:`lib_secure_services`
+
+  * Bluetooth Low Energy:
+
+    * :ref:`bas_c_readme`
+
+  * Other:
+
+    * :ref:`ppi_trace`
+    * :ref:`ble_console_readme`
+
+* Added or updated the following documentation:
+
+  * :ref:`ug_bt_ll_nrfxlib`
+  * :ref:`ug_multi_image`
+  * :ref:`partition_manager`
+  * :ref:`nrf_desktop`
+  * :ref:`shell_bt_nus_readme`
+
+* API documentation of all libraries now also mentions the location of header files and source files.
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :ref:`asset_tracker` sample does not wait for connection to nRF Cloud before trying to send data.
+  This causes the sample to crash if the user toggles one of the switches before the board is connected to the cloud.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
+  Therefore, the LTE modem cannot be operational during debug sessions.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+
+Crypto
+======
+
+* The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+  Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+
+Bootloader
+----------
+
+* Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
+* The immutable bootlader can only be used with the following boards:
+
+  * nrf52840_pca10056
+  * nrf9160_pca10090
+
+DFU
+---
+
+* Firmware upgrade using mcumgr or USB DFU is broken for multi-image builds, because the device tree configuration is not used.
```

Comment:
This subsection should be:
`
```
Firmware upgrade using mcumgr or USB DFU is broken for non-secure applications, because the metadata used by MCUboot is stored in a secure section of flash and is not readable by the non-secure application.
Therefore, it is not possible to upload the image.
To work around this issue, modify mcumgr to hard code the addresses instead of reading them from the metadata.
```

Reply:
done